app/views/enterprise_mailer/welcome.html.haml

```diff
@@ -6,10 +6,10 @@
   = "#{t(:email_registered)} #{ Spree::Config.site_name }!"
 
 %p
-  = t :email_userguide_html, link: link_to('Open Food Network User Guide', ContentConfig.user_guide_link)
+  = t :email_userguide_html, link: link_to(t(:userguide), ContentConfig.user_guide_link)
 
 %p
-  = t :email_admin_html, link: link_to('Admin Panel', spree.admin_url)
+  = t :email_admin_html, link: link_to(t(:admin_panel), spree.admin_url)
```

Comment:
we are migrating al translations to lazy lookup as explained here:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#development

so, in this case, instead of a root level key, we need a relative key
here it needs to be
`= t :email_admin_html, link: link_to(t('.admin_panel'), spree.admin_url)`

and in the en.yml file it needs to be in the specific place of this file
in this case
app/views/enterprise_mailer/welcome.html.haml 
makes a i18n relative  path of: enterprise_mailer.welcome
so this translation needs to be enterprise_mailer.welcome.admin_panel
[here](https://github.com/openfoodfoundation/openfoodnetwork/blob/8a732568e7459bdee8ef07e545c243c7d7689e82/config/locales/en.yml#L138)

Reply:
you could change all the entries in this file (email_welcome, email_registered and join_community) to lazy lookup by replacing :key with '.key' and adding entries in en.yml under enterprise_mailer.welcome

Reply:
> we are migrating al translations to lazy lookup as explained here:
> https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#development
> 
> so, in this case, instead of a root level key, we need a relative key
> here it needs to be
> `= t :email_admin_html, link: link_to(t('.admin_panel'), spree.admin_url)`
> 
> and in the en.yml file it needs to be in the specific place of this file
> in this case
> app/views/enterprise_mailer/welcome.html.haml
> makes a i18n relative path of: enterprise_mailer.welcome
> so this translation needs to be enterprise_mailer.welcome.admin_panel
> [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/8a732568e7459bdee8ef07e545c243c7d7689e82/config/locales/en.yml#L138)

Thanks @luisramos0  for posting the i18n link. :smile: 
I've changed as you suggest, can you take a look?

Reply:
awesome. I just double checked that all these keys are only used in this page, so it's fine to delete them from where they were in the en.yml file :+1:

Reply:
Yeah.. I checked here, especially if `t: email_welcome` could be used for other e-mails. 😊